staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go

```diff
@@ -114,6 +114,11 @@ func transformResponseObject(ctx request.Context, scope RequestScope, req *http.
 				return
 			}
 
+			if scope.TableConvertor == nil {
```

Comment:
We should never get into this code path - this is a programmer error, so we should treat it as such and ensure that it is not possible to enter this code path with a nil table convertor (by tightening earlier checks, not late checking).

Reply:
OK，earlier logic guarantee no nil.   I will close this.